New package: TechI v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: TechI
- Repository: https://github.com/p-w-rs/TechI.jl
- Created by: @p-w-rs
- Version: v0.1.0
- Commit: 0985ba64055b819dff5a36746268ca3ede031809
- Git reference: HEAD
- Description: Technical indicators for algorithmic trading in Julia

goerz commented:
[noblock] You do see the [message posted by the bot](https://github.com/JuliaRegistries/General/pull/120048#issuecomment-2495642434), right?

goerz commented:
Also, probably a better name for this package would be `TechnicalIndicators` (not totally sure if that’s an appropriate technical term, no pun intended). Or maybe `TradingIndicators`. Short, non-descriptive names are generally frowned upon in the Julia community.

JuliaHubRegistrator committed:
New package: TechI v0.1.0

UUID: 89c1805e-7de6-4d5c-a550-370041bb1e4f
Repo: https://github.com/p-w-rs/TechI.jl.git
Tree: 87bf66e9217e76c67d608d2c2ff868b9002d4e16

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/120105
